.github/workflows/develop.yml

```diff
@@ -215,6 +213,11 @@ jobs:
         run: pip install pyarrow
       - name: Install cargo-llvm-cov
         uses: taiki-e/install-action@cargo-llvm-cov
+      - name: Install WSL distribution
+        if: matrix.os == 'windows-latest'
+        uses: Vampire/setup-wsl@v2
```

Comment:
question: Do we need wsl to compile greptimedb on Windows?

Reply:
Actually WSL is already enabled in github windows runner, just lacks of the specific linux distribution, so I install ubuntu here. We need WSL to execute the "fetch-dashboard-assets.sh" script.